examples/example_collect_all_artworks.ipynb

```diff
@@ -42,291 +50,161 @@
-     ]
-    }
-   ],
-   "source": [
-    "!cat ./data/Jan_Van_Der_Heyden/description.txt"
-   ]
-  },
-  {
-   "cell_type": "markdown",
-   "id": "ac2e3754",
-   "metadata": {},
-   "source": [
-    "### Metadata of artist"
-   ]
-  },
-  {
-   "cell_type": "code",
-   "execution_count": 10,
-   "id": "0d0806f7",
-   "metadata": {},
-   "outputs": [
-    {
-     "name": "stdout",
-     "output_type": "stream",
-     "text": [
-      "{\"family name\": \"Van der Heyden\", \"given name\": \"Jan\", \"pseudonym\": \"\", \"sex or gender\": \"male\", \"date of birth\": \"1637-03-05\", \"place of birth\": \"Gorinchem\", \"latitude of place of birth\": \"51.83652\", \"longitude of place of birth\": \"4.97243\", \"date of death\": \"1712-03-28\", \"place of death\": \"Amsterdam\", \"latitude of place of death\": \"52.372777777\", \"longitude of place of death\": \"4.893611111\", \"country of citizenship\": \"Netherlands\", \"residence\": \"\", \"work location\": \"Amsterdam\", \"genre\": \"landscape art\", \"movement\": \"\", \"occupation\": [\"firefighter\", \"inventor\", \"painter\", \"instrument maker\", \"printmaker\"]}"
-     ]
-    }
-   ],
-   "source": [
-    "!cat ./data/Jan_Van_Der_Heyden/metadata.json"
-   ]
-  },
-  {
-   "cell_type": "markdown",
-   "id": "2f128b4b",
+   "execution_count": 4,
+   "id": "bba4ec54",
    "metadata": {},
+   "outputs": [],
    "source": [
-    "### Directory containing works of this artist"
+    "# Find_artworks for each artist\n",
+    "for artist_url in artist_urls:\n",
+    "    with FindArtworks(artist_link=artist_url, output_dir=output_dir, \n",
+    "                      min_wait_time=min_wait_time) as scraper:\n",
+    "            # Save list of artworks, the description, and metadata for an artist\n",
+    "            retry(scraper.save_artist_information, max_retries, min_wait_time)\n",
+    "\n",
+    "            # Find artist directory\n",
+    "            artist_dir = output_dir + '/' + scraper.get_artist_name() \n",
+    "\n",
+    "                \n",
+    "    # Scrape artworks\n",
+    "    with GoogleArtScraper(artist_dir + '/' + 'works', min_wait=min_wait_time) as subscraper:\n",
+    "        # Get list of links to this artist's works \n",
+    "        with open(artist_dir+'/'+'works.txt', 'r') as file:\n",
+    "            artwork_links = [line.rstrip() for line in file]  \n",
+    "            \n",
+    "        # Download all artwork link (slow)\n",
+    "        for url in artwork_links:\n",
+    "            print(f'artwork URL: {url}')\n",
+    "            retry(subscraper.save_artwork_information, max_retries, min_wait_time, url)"
    ]
   },
   {
    "cell_type": "code",
-   "execution_count": null,
-   "id": "54afc420",
+   "execution_count": 5,
```

Comment:
Cell 5 is empty and needs to be removed.

Reply:
Yes please